lms/templates/courseware/course_about.html

```diff
@@ -324,6 +324,11 @@ <h2 class="opencourseware">MITOpenCourseware</h2>
     </div>
       %endif
 
+    % if get_course_about_section(request, course, "about_sidebar_html"):
+      <section class="about-sidebar-html">
```

Comment:
while the test does not indicate this is the issue, we do have a custom rule that looks to see if every section element has a heading as its first child (which the HTML5 spec recommends).  It could be failing because, well, the content of the section is not known here as its a var.  It could contain heading, it could not, but its not a hard requirement.

Reply:
Is it ok if I leave this as is?